src/transformStateWithClones.js

```diff
@@ -6,8 +6,32 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
-  // write code here
-}
+  let currentState = { ...state };
```

Comment:
The name 'currentState' is not fully indicative of its purpose. A name like 'stateClone' or 'stateSnapshot' would be more descriptive.